Applications/AdminCli/src/AdminCli/Commands/Announcements/AnnouncementCommand.cs

```diff
@@ -0,0 +1,134 @@
+        };
+
+        AddOption(expiresAt);
+        AddOption(severity);
+
+        this.SetHandler(SendAnnouncement, DB_PROVIDER_OPTION, DB_CONNECTION_STRING_OPTION, severity, expiresAt);
+    }
+
+    private async Task SendAnnouncement(string dbProvider, string dbConnectionString, string? severityInput, string? expiresAtInput)
+    {
+        try
+        {
+            var severity = severityInput switch
+            {
+                _ when string.IsNullOrWhiteSpace(severityInput) => AnnouncementSeverity.Low,
+                _ when Enum.TryParse<AnnouncementSeverity>(severityInput, ignoreCase: true, out var parsedSeverity) => parsedSeverity,
+                _ => throw new ArgumentException($@"Specified severity '{severityInput}' is not a valid severity.")
+            };
+
+            DateTime? expiresAt = expiresAtInput switch
+            {
+                _ when string.IsNullOrWhiteSpace(expiresAtInput) => null,
+                _ when DateTime.TryParse(expiresAtInput, out var parsedDateTime) => parsedDateTime,
+                _ => throw new ArgumentException($@"Specified expiration datetime '{expiresAtInput}' is not a valid DateTime.")
+            };
+
+            var texts = ReadTextsFromCommandLineInput();
+
+            if (texts.Count == 0)
+            {
+                Console.WriteLine(@"No texts provided. Exiting...");
+                return;
+            }
+
+            Console.WriteLine(@"Sending announcement...");
+
+            var mediator = _serviceLocator.GetService<IMediator>(dbProvider, dbConnectionString);
+
+            var response = await mediator.Send(new CreateAnnouncementCommand
+            {
+                Texts = texts,
+                Severity = severity,
+                ExpiresAt = expiresAt
+            }, CancellationToken.None);
+
+            Console.WriteLine(@"Announcement sent successfully");
+            Console.WriteLine(JsonSerializer.Serialize(response, JSON_SERIALIZER_OPTIONS));
+        }
+        catch (Exception e)
+        {
+            Console.WriteLine($@"An error occurred: {e.Message}");
+        }
+    }
+
+    private static List<CreateAnnouncementCommandText> ReadTextsFromCommandLineInput()
+    {
+        var texts = new List<CreateAnnouncementCommandText>();
+        bool addAnotherLanguage;
+        do
+        {
+            var language = PromptForInput(@"Enter language (e.g. en, de, it, nl). At least 1 must be in english: ");
+            var title = PromptForInput(@"Enter title: ");
+            var body = PromptForInput(@"Enter body: ");
+
+            if (language == null || title == null || body == null)
+            {
+                break;
+            }
+
+            texts.Add(new CreateAnnouncementCommandText
+            {
+                Language = language,
+                Title = title,
+                Body = body
+            });
+
+            var input = PromptForInput(@"Do you want to add another language? ([y]es/[n]o): ");
```

Comment:
We should probably capitalize the "N" of "[n]o" to make it clear that it's the default option.
```suggestion
            var input = PromptForInput(@"Do you want to add another language? ([y]es/[N]o): ");
```

Reply:
Ok